give/src/components/ComponentTemplate.js

```diff
@@ -1,24 +1,28 @@
+import { ComponentWrapper } from "./shared";
 // import "./Component.css"; // Import component stylesheet
 
 export const Component = ({
   className = "", // string
-  str = "Hello world!", // string
-  bool = true, // bool
+  children,
 }) => {
-  if (!bool) return null; // Condition for rendering nothing
+  if (!children) return null; // Condition for rendering nothing
 
-  const classes = className !== "" ? " " + className : className;
-
-  return <p className={"component-class" + classes}>{str}</p>;
+  return (
+    <ComponentWrapper type="test" className={className}>
+      {children}
+    </ComponentWrapper>
+  );
 };
 
 export const ComponentExamples = () => {
   return (
     <>
-      <Component />
-      <Component className="someClass" />
-      <Component str={"New text"} />
-      <Component str={"Hidden text"} bool={false} />
+      <Component className="someClass">
+        <p>
+          This is a simple component to be used as a template for building out
+          larger components.
+        </p>
```

Comment:
Is this only a wrapper? Can't really tell how it is a template?

Reply:
This is a template in the sense that it gives me a starting point for how to organize my code when building out new components. I try to be as consistent in my code as possible and this helps. It won't actually be used in the final project. Maybe I should call it something different.

Reply:
@j2trumpet Ah so something like bolierplate?

Reply:
@LizzyRandall Something like that.